engine/polyglot-api/src/main/java/org/enso/polyglot/RuntimeOptions.java

```diff
@@ -26,6 +26,11 @@ public class RuntimeOptions {
   private static final OptionDescriptor DISABLE_PRIVATE_CHECK_DESCRIPTOR =
       OptionDescriptor.newBuilder(DISABLE_PRIVATE_CHECK_KEY, DISABLE_PRIVATE_CHECK).build();
 
+  public static final String ENABLE_TYPE_CHECK = optionName("enableTypeCheck");
```

Comment:
We can run with typechecker enabled by passing the `--enable-type-check` flag to the runner.

By default it is disabled.

Reply:
I renamed the option to `--enable-static-analysis` in 9fccb7e6

Do you think that sounds good or do you propose some other name for this option?